public/js/pimcore/element/abstract.js

```diff
@@ -40,6 +40,10 @@ pimcore.element.abstract = Class.create({
 
     addToHistory: true,
 
+    checkIfNewHeadbarLayoutIsEnabled: function () {
+        return pimcore?.settings?.new_admin_style;
```

Comment:
```suggestion
        return pimcore.settings?.new_admin_style;
```
`pimcore.settings` are always set in index action

Reply:
Done.